slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -61,6 +67,8 @@ trait OracleProfile extends JdbcProfile {
 
   override protected def computeCapabilities: Set[Capability] = (super.computeCapabilities
     - RelationalCapabilities.foreignKeyActions
+    - RelationalCapabilities.createIfNotExists
+    - RelationalCapabilities.dropIfExists
```

Comment:
I am confused. It looks like you added this feature to Oracle, but this says Oracle doesn't support this. Is this a mistake?